### PR TITLE
Fix possibly incorrect `chown` on /usr/local

### PR DESCRIPTION
**Pull request by AravK**
Should be on CREW_PREFIX

### CODE REVIEW

**Comment by uberhacker** [comment]
Good call!

File: install.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ case "$architecture" in
 esac
 
 #This will allow things to work without sudo
-sudo chown -R `id -u`:`id -g` /usr/local
+sudo chown -R `id -u`:`id -g` "${CREW_PREFIX}"
```
Comment (uberhacker) [inline install.sh]: Should be `"#{CREW_PREFIX}"`.

Comment (jam7) [reply install.sh]: it's a shell script.  not a ruby script.

**Comment by jam7** [comment]
Thank you for your PR.

File: install.sh (hunk 1 above)
Comment (uberhacker) [reply install.sh]: Oops, missed that.

**Pull request merged**